packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -1,17 +1,61 @@
 import { Direction } from "@microsoft/fast-application-utilities";
+import { memoize } from "lodash-es";
 
 export interface IDesignSystem {
+    /**
+     * The value typically used for foreground elements, such as text
+     */
     foregroundColor: string;
+
+    /**
+     * The value typically used for backgrounds of elements
+     */
     backgroundColor: string;
+
+    /**
+     * The brand color. This value is deprecated in lieu of accentColor
+     * @deprecated
+     *
+     */
     brandColor: string;
+
+    /**
+     * The primary direction of the document

```

Comment:
do we want to say document? because you could wrap anything with the design system